features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/explore/SearchFragment.kt

```diff
@@ -554,19 +563,12 @@ class SearchFragment : Fragment(R.layout.fragment_search) {
             }
         }
 
-        requireActivity()
-            .onBackPressedDispatcher
-            .addCallback(
-                viewLifecycleOwner,
-                object : OnBackPressedCallback(true) {
-                    override fun handleOnBackPressed() {
-                        hardBackAction.invoke()
-                        if (isMerchantTopic) {
-                            viewModel.logEvent(AnalyticsConstants.Explore.MERCHANT_DETAILS_BACK_BOTTOM)
-                        }
-                    }
-                }
-            )
+        onBackPressedCallback = requireActivity().onBackPressedDispatcher.addCallback(viewLifecycleOwner) {
+            hardBackAction.invoke()
+            if (isMerchantTopic) {
+                viewModel.logEvent(AnalyticsConstants.Explore.MERCHANT_DETAILS_BACK_BOTTOM)
+            }
+        }
```

Comment:
code was simplified to be more kotlin than Java.  Docs were chanced to ensure that the parameter set to to`true` would be `true` in kotlin which uses a default value for the parameter.

I will need to make these changes separately to the Maya integration.